company.el

```diff
@@ -3218,10 +3218,10 @@ If SHOW-VERSION is non-nil, show the version in the echo area."
                   'company-scrollbar-fg
                 'company-scrollbar-bg)))
 
-(defun company--scrollpos-line (text width right-margin-width)
```

Comment:
Oh, right.

But could you rename it to e.g. `fancy-margin-width`, please?

Because "margin width", in general, is covered by the variable `company-tooltip-margin`.

Reply:
done

Reply:
Thanks!